### PR TITLE
New: TreePicker helptips for average/sum methods

### PR DESCRIPTION
**Pull request by camjc**
#### What does this PR accomplish?
TreePicker helptips for average/sum methods

#### How should this be manually tested?
–

#### What are the relevant trello cards?
https://trello.com/c/MsN6bc7D/5000-update-treepicker-helptip-for-subtotal-average

#### Screenshots (if appropriate)

screen shot 2016-02-09 at 11 30 25 am

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

File: src/components/Main.js [hunk 1]
```diff
@@ -305,6 +305,12 @@ class AppComponent extends React.Component {
           baseItem={baseItem}
           modalClose={this.toggleTreePickerModal}
           initialSelection={initialSelection}
+          helpText={{
+            average: `CPM line items use the average method.
+              Targeting adjusts which people, but not the number of people who see your ad.`,
```
Comment (pphminions) [inline src/components/Main.js]: Just wondering if the quote character should be ' not ` :)

Comment (pphminions) [inline src/components/Main.js]: LGTM :leaves:

**Comment by pphminions** [comment]
LGTM :leaves:

File: src/components/Main.js (hunk 1 above)
Comment (camjc) [inline src/components/Main.js]: the ` lets you do multi line or template strings in ES2015.

Comment (pphminions) [inline src/components/Main.js]: thanks for the explanation Cam :) (sorry about the question :) )

**Comment by omgaz** [comment]
LGTMerging :crocodile:

**Pull request merged**